beluga_amcl/README.md

```diff
@@ -1,7 +1,7 @@
 # Beluga AMCL
 
-Beluga AMCL is a ROS 2 node based on the [Beluga](../beluga) library that aims to be fully compatible with the [Navigation 2 AMCL][nav2_amcl] node.<br/>
-The compatibility between `beluga_amcl` and `nav2_amcl` provides a simple migration path for projects that want to be able to integrate the power and modularity of the Beluga library in an existing `nav2_amcl`-based project.
+Beluga AMCL is a ROS node based on the [Beluga](../beluga) library that aims to be fully compatible with both [Navigation 2 AMCL][nav2_amcl] and [Navigation AMCL][amcl] nodes.<br/>
+The compatibility between `beluga_amcl` and its longstanding counterparts in the ROS ecosystem provides a simple migration path for projects that want to be able to integrate the power and modularity of the Beluga library in an existing `nav2_amcl`-based (or `amcl`-based) project.
 
 ## ROS 2 Interface
```

Comment:
nit: A table of contents here would be super useful.

Reply:
Done in https://github.com/Ekumen-OS/beluga/pull/218/commits/2c6130aad012ca003cf1af8e863910f6d2a21ff0.